backend/app/app/api/endpoints/jobs.py

```diff
@@ -14,10 +14,25 @@
 def list_jobs(
         db: Session = Depends(get_db),
         skip: int = 0,
-        limit: int = 100,
+        limit: int = 50,
 ):
-    """List jobs"""
-    jobs = jobs_service.get_items(db, skip=skip, limit=limit)
+    """List jobs - page 0"""
+    return list_job_page(db, 0, limit)
+
+
+@router.get("/page/{page}", response_model=List[Job])
+def list_job_page(
+        db: Session = Depends(get_db),
+        page: int = 0,
+        limit: int = 50,
+):
+    """List jobs by page"""
+    skip = page * limit
```

Comment:
If jobs get added while a client is paginating through the results would things get missed? It seems like a since field would work here (either by date or some other orderable field)

Reply:
That's correct, but the risk seems extremely low. You would miss a job if all of the following occur at once:
1) the job you are looking for is nth job on a page where n==limit (1/limit chance of occurring, in this case 1/50),
2) you are searching forward in time rather than back in time (probably less than 1/2 chance of occurring, since going back will probably be far more common)
3) a new job starts between when your last page data refresh and when you go to the next page (very unlikely since we bake about 0.003 books per refresh)
4) your current page is the one right after the page you would find your job on (some variable chance)